samples/boards/reel_board/mesh_badge/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _mesh_badge:
+
+Mesh Badge
+##########
+
+Overview
+********
+
+A sample app for the reel board that showcases Bluetooth Mesh
+
+The app starts off as a regular Bluetooth GATT peripheral application. It's
```

Comment:
Delete ``It's recommended to" and start the sentence with:

    Install the ...

Reply:
done